.circleci/config.yml

```diff
@@ -291,6 +291,7 @@ workflows:
           filters: *tag_filters
           requires:
             - build_artifacts
+          context: Honeycomb Secrets for Public Repos
```

Comment:
With the added to this `publish_rubygems` YAML key that is aliased as `&publish`, the `context:` entry could be removed from the following `publish_github` section which includes all of `&publish`.

Reply:
ooh good eye! https://github.com/honeycombio/beeline-ruby/pull/219/commits/d44b676a5ed952ac1ee6e401952962798f3f7065